dracut/modules.d/80kiwi-verity/Makefile

```diff
@@ -0,0 +1,26 @@
+buildroot ?= /
+
+CROSS_COMPILE ?=
+CC ?= $(CROSS_COMPILE)gcc
+STRIP ?= $(CROSS_COMPILE)strip
```

Comment:
We should probably not have this here, since stripping is controlled by the build environment.